lib/new_relic/agent/instrumentation/active_record_helper.rb

```diff
@@ -229,11 +232,16 @@ module InstanceIdentification
           SLASH = "/".freeze
           LOCALHOST = "localhost".freeze
 
+          def adapter_from_config(config)
+            bare_name = NewRelic::Agent::Instrumentation::ActiveRecordHelper.bare_adapter_name(config[:adapter])
```

Comment:
We're currently in the `InstanceIdentification` namespace here, so explicitly back up a level to get at `bare_adapter_name`.